app/src/main/res/values/dimens.xml

```diff
@@ -2,4 +2,7 @@
     <!-- Default screen margins, per the Android Design guidelines. -->
     <dimen name="activity_horizontal_margin">16dp</dimen>
     <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
+    <dimen name="contest_app_textsize">36sp</dimen>
```

Comment:
Maybe we can standardize the name (contest_app**_textsize** vs sub_header**_size**)